bin/p2-bootstrap/bootstrap.go

```diff
@@ -37,6 +37,7 @@ func main() {
 			log.Fatalf("Could not get consul manifest: %s", err)
 		}
 		consulPod := pods.PodFromManifestId(consulManifest.ID())
+		consulPod.RunAs = "root"
```

Comment:
I had to make the `RunAs` for a pod configurable, which had the nice effect of removing the hardcoded pod names that were present in the pod package.